Platform/AlderlakeBoardPkg/BoardConfigAdlN50.py

```diff
@@ -0,0 +1,490 @@
+## @file
+# This file is used to provide board specific image information.
+#
+#  Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+##
+# Import Modules
+#
+import os
+import sys
+import time
+
+sys.dont_write_bytecode = True
+sys.path.append (os.path.join('..', '..'))
+from BuildLoader import *
```

Comment:
SBL supports board overridden. It would be good to use that to give another example on how to support ADLN50 board based on ADLN to provide necessary override instead providing a full config file.

Reply:
hi guo, 

this boardconfigadln50.py file is newly created because the bios size for adln50 is relatively small compared to the normal adln, so the size is adjusted to match the bios size (around 10KB). 

if using the boardconfigadln.py, later during the stitching part would get error as we can't auto-adjusted the bios region size to fit in the binary.

Reply:
I mean you could to use boardconfigadln50.py, and let boardconfigadln50.py based on boardconfigadln.py to only have overridden items to save hundreds of line code.

Reply:
hi @gdong1 , amended the boardconfigadln50.py to be configured based on boardconfigadln.py, indeed saving lots of lines, thank you for you input. Tested and verified the latest changes is bootable to both ubuntu & windows.

Reply:
Thanks for the update. Please fix the patch check failures.
From failure logs as below, it looks there are spaces in empty line
Trailing whitespace found
   File: Platform/AlderlakeBoardPkg/BoardConfigAdlN50.py
   Line:         
 * Trailing whitespace found
   File: Platform/AlderlakeBoardPkg/BoardConfigAdlN50.py
   Line: